packages/authentication/src/migrations/Migration20240115092929.ts

```diff
@@ -0,0 +1,15 @@
+import { Migration } from '@mikro-orm/migrations';
```

Comment:
comment: You should be able to omit the migration entirely until we have finalized the module. The tests will read the models and do not depend on migration files.